<!-- Launch file to move on a preprogrammed trajectory -->

<launch>
	<group ns="saf">
	
		<arg name="rate" default="10"/>
  	
			<node name="dummy_vision" pkg="irob_vision_support" 
			type="vision_server_test_dummy" output="screen">
  		
  				<param name="rate" 
						type="double" value="$(arg rate)" />
						
				<remap from="left/color_image" 
							to="stereo/left/image_rect_color" />				
				<remap from="right/color_image" 
							to="stereo/right/image_rect_color" />
										
				<remap from="left/image" 
							to="stereo/left/image_rect" />			
				<remap from="right/image" 
							to="stereo/right/image_rect" />
										
				<remap from="disparity" 
							to="stereo/disparity" />
							
				<remap from="result" 
							to="vision/target" />
							
				<remap from="marker" 
							to="vision/dummy_target_marker" />
					
					
		
	</node>
  	</group>
</launch>
